repo-validation/test-mit/SECURITY.md

```
@@ -0,0 +1,30 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability within this project, please send an email to security@example.com. All security vulnerabilities will be promptly addressed.
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)